src/Application/Game.cpp

```diff
@@ -1726,16 +1777,23 @@ void Game::EventLoop() {
                         GameUI_SetStatusBar(LSTR_CANT_REST_IN_TURN_BASED);
                         continue;
                     }
-                    if (!(pParty->uFlags & 0x88)) {
+
+                    if (!(pParty->uFlags & (PARTY_FLAGS_1_AIRBORNE | PARTY_FLAGS_1_STANDING_ON_WATER))) {
                         pGUIWindow_CurrentMenu = new GUIWindow_Rest();
                         continue;
+                    } else {
+                        if (engine->config->verbose_logging) {
+                            if (pParty->uFlags & PARTY_FLAGS_1_AIRBORNE) logger->Info("Party is aribourne");
```

Comment:
"aribourne" -> "airborne"?

Reply:
Speaking of spellings in general I would also prefer to have US spellings over UK ones. For example initialization and not initialisation, color and not colour and so on.